wandb/__init__.py

```diff
@@ -456,6 +464,26 @@ def _get_python_type():
         return "python"
 
 
+def parse_sm_config():
+    sagemaker_config = "/opt/ml/input/config/hyperparameters.json"
+    if os.path.exists(sagemaker_config):
+        conf = {}
+        # Hyper-parameter searchs quote configs...
+        for k, v in six.iteritems(json.load(open(sagemaker_config))):
```

Comment:
Can the sagemaker config have nested objects and arrays? (In which case we'd want to do this recursively?)

Reply:
I don't think they can, but they are doing a weird json encoding thing I wish I understood.